composer.json

```diff
@@ -16,19 +16,22 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": "^5.6 || ^7.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/intl": "^2.8 || ^3.0 || ^4.0",
-        "symfony/templating": "^2.8 || ^3.0 || ^4.0",
-        "symfony/validator": "^2.8 || ^3.0 || ^4.0",
-        "symfony/yaml": "^2.8 || ^3.0 || ^4.0",
-        "psr/log": "~1.0"
+        "php": "^7.2 || ^8.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/intl": "^4.3 || ^5.0",
+        "symfony/templating": "^4.3 || ^5.0",
+        "symfony/validator": "^4.3 || ^5.0",
+        "symfony/yaml": "^4.3 || ^5.0",
```

Comment:
Can you please allow Symfony 6 as well? Or is there additional work necessary for it to work with Symfony 6?

Reply:
I think upgrading to symfony 6 would be quite easy, but didn't have the time for this yet. Maybe also better to do this in a separate MR ;)